content/spaces/backup-and-restore.md

```diff
@@ -0,0 +1,189 @@
+
+## Usage
+
+### Backup
+
+#### Create a backup secret
+
+Before you begin the backup process, you need to create a secret for your Kubernetes configuration to allow for communication between your backup and your restored configuration.
+
+```bash
+KUBECONFIG=/tmp/ctp.yaml kubectl create secret generic super-secret-secret -n default --from-literal=password=supersecret
+```
+
+#### Single backup
+
+You can create a backup of your current ControlPlane.
+
+```yaml
+apiVersion: spaces.upbound.io/v1alpha1
+kind: Backup
+metadata:
+  name: my-awesome-ctp-backup
+  namespace: default
+spec:
+  controlPlane: my-awesome-ctp
+  deletionPolicy: Delete
+```
+
+Once your ControlPlane indicates the backup is complete, you can delete the ControlPlane:
+
+```bash
+kubectl wait backup my-awesome-ctp-backup --for condition=Completed=True --timeout=3600s && \
+kubectl delete controlplane my-awesome-ctp
+```
+
+#### Shared backup
+
+You can define the `SharedBackupConfig` for multiple control planes in a configuration file like the example below:
+
+```yaml
+apiVersion: spaces.upbound.io/v1alpha1
+kind: SharedBackupConfig
+metadata:
+  name: default
+  namespace: default
+spec:
+  objectStorage:
+    provider: AWS
+    bucket: philippe-scorsolini-spaces-backup-restore-test
+    config:
+      endpoint: s3.eu-west-2.amazonaws.com
+      region: eu-west-2
+    credentials:
+      source: Secret
+      secretRef:
+        name: bucket-creds
+        key: creds
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bucket-creds
+  namespace: default
+stringData:
+  creds: |
+    [default]
+    aws_access_key_id=***************
+    aws_secret_access_key=*************************
+```
+
+Then create a `ControlPlane` configuration with that secret reference:
+
+```yaml
+apiVersion: spaces.upbound.io/v1beta1
+kind: ControlPlane
+metadata:
+  labels:
+    org: foo
+  name: my-awesome-ctp
+  namespace: default
+spec:
+  backup:
+    sharedBackupConfigRef:
+      name: default
+  writeConnectionSecretToRef:
+    name: kubeconfig-my-awesome-ctp
+---
+apiVersion: spaces.upbound.io/v1beta1
+kind: ControlPlane
+metadata:
+  labels:
+    org: foo
+  name: my-second-awesome-ctp
+  namespace: default
+spec:
+  backup:
+    sharedBackupConfigRef:
+      name: default
+  writeConnectionSecretToRef:
+    name: kubeconfig-my-second-awesome-ctp
+```
+
+Then a `SharedBackup` configuration:
+
+```yaml
+apiVersion: spaces.upbound.io/v1alpha1
+kind: SharedBackup
+metadata:
+  name: custom-shared-backup
+  namespace: default
+spec:
+  controlPlaneSelector:
+    labelSelectors:
+    - matchLabels:
+        org: foo
+```
+
+This example configuration results in both control planes backed up and uploaded to the S3 endpoint.
+
+### Create a scheduled backup
+
+Instead of a one-time backup with the `SharedBackup` kind, you can create a `SharedBackupSchedule` configuration:
```

Comment:
```suggestion
Instead of a one-time backup with the `SharedBackup` kind, you can create a `SharedBackupSchedule:
```

instead of calling these `configurations` (heavily overloaded term in our space due to Crossplane Configurations), what do you think about referring to these as resources, objects, or just dropping configuration entirely? Think it reads just as fine.